File: mvn-advent-of-code/src/test/java/advent/twenty_fifteen/Day25LetItSnowTest.java
package advent.twenty_fifteen;

import advent.common.DailyProblem;
import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class Day25LetItSnowTest {
    @Test
    public void checkReal() {
        DailyProblem<Long, Long> dp = new Day25LetItSnow("/twenty_fifteen/Day25-LetItSnow-input.txt");

        assertEquals(8997277, dp.getPart1Answer().longValue());
    }

    @Test
    public void checkExample1() {
        DailyProblem<Long, Long> dp = new Day25LetItSnow("/twenty_fifteen/Day25-LetItSnow-example#1.txt");

        assertEquals(33511524, dp.getPart1Answer().longValue());
    }
}

File: scala-advent-of-code/src/test/scala/advent/twenty_fifteen/Day25LetItSnowTest.scala
package advent.twenty_fifteen

import org.scalatest.{FlatSpec, Matchers}

class Day25LetItSnowTest  extends FlatSpec with Matchers {
  "2015 Day 25 - Input" should "calculate answers" in {
    val dp = new Day25LetItSnow("/twenty_fifteen/Day25-LetItSnow-input.txt")
    dp.part1Answer should be(8997277)
  }

  "2015 Day 9 - Example #1" should "calculate answers" in {
    val dp = new Day25LetItSnow("/twenty_fifteen/Day25-LetItSnow-example#1.txt")
    dp.part1Answer should be(33511524)
  }
}
